Content.Client/SurveillanceCamera/UI/SurveillanceCameraMonitorBoundUi.cs

```diff
@@ -0,0 +1,123 @@
+using Content.Client.Eye;
+using Content.Shared.SurveillanceCamera;
+using Robust.Client.GameObjects;
+
+namespace Content.Client.SurveillanceCamera.UI;
+
+public sealed class SurveillanceCameraMonitorBoundUserInterface : BoundUserInterface
+{
+    [Dependency] private readonly IEntityManager _entityManager = default!;
+    private readonly EyeLerpingSystem _eyeLerpingSystem = default!;
+    private readonly SurveillanceCameraMonitorSystem _surveillanceCameraMonitorSystem;
+
+    private SurveillanceCameraMonitorWindow? _window;
+    private EntityUid? _currentCamera;
+
+    public SurveillanceCameraMonitorBoundUserInterface(ClientUserInterfaceComponent owner, object uiKey) : base(owner, uiKey)
+    {
+        IoCManager.InjectDependencies(this);
+        _eyeLerpingSystem = EntitySystem.Get<EyeLerpingSystem>();
+        _surveillanceCameraMonitorSystem = EntitySystem.Get<SurveillanceCameraMonitorSystem>();
```

Comment:
EntitySystem.Get<T> is deprecated so just use something like _entityManager.EntitySysManager.GetEntitySystem<T>